sigs.yaml

```diff
@@ -160,9 +160,6 @@ sigs:
     - github: kow3ns
       name: Kenneth Owens
       company: Brex
-    - github: mattfarina
-      name: Matt Farina
-      company: Rancher Labs
```

Comment:
Thank you @mattfarina for everything <3
One thought - (you don't have to of course), but would you like to add yourself as emeritus?
https://github.com/kubernetes/community/blob/1248937700a634e36bd2548b259d755b7e7af8b9/sigs.yaml#L169-L171

Reply:
doh, missed that. I'll fix it.

Reply:
fixed